core/common/src/MapLayerSettings.ts

```diff
@@ -190,13 +190,27 @@ export interface ImageMapLayerProps extends CommonMapLayerProps {
 
 }
 
+/** The target onto which to drape a model map layer.
```

Comment:
Is it not possible to drape a model onto both reality data and ellipsoid? Would you have to create two otherwise-identical model map layers with different drape targets?